optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -482,45 +501,23 @@ def _from_pretrained(
     def _from_transformers(
         cls,
         model_id: str,
-        subfolder: Optional[str] = "",
+        config: "PretrainedConfig",
         save_dir: Union[str, Path] = default_cache_path,
-        use_auth_token: Optional[Union[bool, str, None]] = None,
-        revision: Optional[Union[str, None]] = None,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        revision: Optional[Union[str]] = None,
         force_download: bool = True,
         cache_dir: Optional[str] = None,
-        **kwargs,
+        subfolder: str = "",
+        local_files_only: bool = False,
+        use_cache: bool = True,
+        provider: str = "CPUExecutionProvider",
```

Comment:
It looks like `use_auth_token`, `revision`, `local_files_only` parameters are not used